FreeRTOS/FreeRTOS-Plus-TCP
# Ut build fix ip ipv4

Ut build fix IP ipv4

Description
-----------
Fixing build issues for IP unit tests build issues

Test Steps
-----------
Unit tests builds

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
